cpp/src/arrow/json/reader_test.cc

```diff
@@ -274,5 +275,37 @@ TEST(ReaderTest, ListArrayWithFewValues) {
   AssertTablesEqual(*actual_table, *expected_table);
 }
 
+TEST_P(ReaderTest, UnquotedDecimal) {
```

Comment:
Can you move these tests near the other parameterized (TEST_P) tests?

Reply:
done